packages/webapp-libs/webapp-finances/src/components/stripe/stripePaymentMethodSelector/methods/newCardInput/newCardInput.styles.ts

```diff
@@ -13,7 +12,7 @@ export const StripeIframeStyles = (theme: Themes) => ({
     border: `1px solid red`,
     padding: '10px',
   },
-  invalid: { color: color.input.invalid, iconColor: color.input.invalid },
+  invalid: { color: '#ef4444', iconColor: '#ef4444' },
```

Comment:
WDYF to use tailwind base colors like `red-500` for invalid input

Reply:
It didn't work :/ 
<img width="411" alt="image" src="https://github.com/apptension/saas-boilerplate/assets/116499136/928d7dd7-79c4-4ca0-bac4-416a66ef4e0d">

Reply:
I think you need to rewrite the entire StripeIframeStyles function like the StripeIframeClasses object, because it depends on the theme from styled components.

red-500 to tylko kolor, aby zmienić kolor czcionki należy użyć klasy tailwind "text-red-500".

Reply:
I think we can leave the object with styles for now and decide later what to do